shopinvader_customer_multi_user/tests/test_multi_user_token.py

```diff
@@ -19,17 +21,17 @@ def _generate_random_companies(self, count=5):
                 name="ACME %s" % x,
                 email="acme%s@test.com" % x,
                 is_company=True,
+                external_id="acme%s" % (x + random.randint(1, 100)),
```

Comment:
This could potentially yield two equal external_id

ie:
- `x = 1` + `randint = 5`
- `x = 2` + `randint = 4` 

both resulting in `acme6` external_id.
Not sure if having duplicate external ids is a real issue, but I'm guessing it is
I'm wondering what's wrong with just doing this:

```suggestion
                external_id="acme%s" % x,
```

Alternatively if you really need random external_ids, maybe try this:

```suggestion
                external_id="acme%s-%s" % (x, random.randint(1, 100)),
```